riot/lol/match.go

```diff
@@ -14,6 +14,35 @@ type MatchClient struct {
 	c *internal.Client
 }
 
+// MatchListOptions
+type MatchListOptions struct {
+	// Set of champion IDs for filtering the matchlist.
+	Champion int
+	// Set of queue IDs for filtering the matchlist.
+	Queue int
+	// The begin time to use for filtering matchlist specified as epoch milliseconds.
+	BeginTime int64
+	// The end time to use for filtering matchlist specified as epoch milliseconds.
+	EndTime int64
```

Comment:
For ease of use, I think it makes sense to let the user pass in a `time.Time` value here and for `BeginTime` and convert it in `buildParam`.